best-practices/native/nativescript/evironment.md

```diff
@@ -0,0 +1,76 @@
+The best practice recommends not use more than 2 properties, production and apiUrl properties.
```

Comment:
A link would be nice for this explanation.

Reply:
Done.